src/styles/_catalog.scss

```diff
@@ -1,22 +1,45 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 48px;
 
   max-width: 944px;
+
   margin: 0 auto;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+
     height: 300px;
+    width: 200px;
+
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
```

Comment:
this is the repetition of styles. You use the same propery a lot of times. Consider how to do it with a variable. Maybe you should consider a mobile-first approach